README.md

```diff
@@ -105,7 +105,7 @@ You can add splitters to any part of your build stream. We recommend using them
 
 #### project.bundler()
 
-A stream that combines the files in your application to reduce the number of frontend requests needed. This can be a great way to [improve performance](https://developer.yahoo.com/performance/rules.html#num_http) when HTTP2/Push is not available.
+A stream that combines the files in your application to reduce the number of frontend requests needed. This can be a great way to [improve performance](https://developer.yahoo.com/performance/rules.html#num_http) when HTTP/2 Push is not available.
```

Comment:
I'd remove the "when..." part, since our own experiments show that bundling is always a big win.